spes.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Anar Jabiyev (Jabiyeff)",
+        "oem": "Redmi",
+        "device": "Redmi Note 11",
+        "filename": "crDroidAndroid-13.0-20221106-spes-v9.0.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/spes/9.x/crDroidAndroid-13.0-20221106-spes-v9.0.zip/download",
+        "timestamp": 1667722116,
+        "md5": "d5afbd87714c27fd594beb52f1ecf5b5",
+        "sha256": "a2ed203c4a40b96c315049f6943028f585469599951d43f41ca0fc554fb37e89",
+        "size": 1199972504,
+        "version": "9.0",
+        "buildtype": "Weekly",
+        "forum": "https://forum.xda-developers.com/t/rom-official-12-1-crdroidandroid-8-x-redmi-note-11-spes-spesn.4440747",
```

Comment:
Update forum info

Reply:
Will do

Reply:
> Update forum info

Done

Reply:
set changelog and download page url to website please
dl: https://crdroid.net/spes/9
changelog: https://crdroid.net/spes/9#changelog

Reply:
Sure, I already using download link from crDroid website on telegram releases. Also fixed now on XDA forum.